tests/src/ut_test_kvp.c

```diff
@@ -130,6 +140,159 @@ void test_ut_kvp_open( void )
     ut_kvp_close(pInstance);
 }
 
+int read_file(const char* filename, test_ut_memory_t* pInstance)
+{
+    FILE *file = fopen(filename, "r+");
+    if (file == NULL)
+    {
+        UT_LOG_ERROR("fopen");
+        pInstance->length = 0;
+        return -1;
+    }
+
+    // Negative Offset: this check should fail for zero length file
+    if (fseek(file, -1, SEEK_END) != 0)
```

Comment:
I would have just checked the length... But it's valid to open a zero length file, you need to be able to-do that during the testing.

Reply:
Bit confused, are you suggesting any change here?? or its ok this way??

Reply:
file moved to ut-control